### DISCUSSION TITLE
Is it possible to get the same image in Fooocus as in A1111?

### DISCUSSION THREAD

**Discussion by ItsCrea** [lllyasviel/Fooocus#1943] [Q&A]
Decided to try to generate the same image in Fooocus and in A1111. The model, prompt, seed, sampling steps are identical. Offset Lora is disabled. CFG is the same. Sharpness in Fooocus is 0. The result is different images. 
What else makes the difference? And is it even possible to achieve the same result?

**Comment by poor7** [comment]
I think it's impossible. Even in one program it can be difficult to get a similar picture.

**Comment by mirek190** [comment]
> Decided to try to generate the same image in Fooocus and in A1111. The model, prompt, seed, sampling steps are identical. Offset Lora is disabled. CFG is the same. Sharpness in Fooocus is 0. The result is different images. What else makes the difference? And is it even possible to achieve the same result?

Different implementations of scheduler , latent,  etc. No possible to get same results.
